### DISCUSSION TITLE
Unable to Join Any Node to New Master Install

### DISCUSSION THREAD

**Discussion by imalc3142** [Security-Onion-Solutions/securityonion#8642] [Unsupported Versions]
I have a new SO installed (2.3.150).  Master installs fine when I try to join a node, any node in a separate master node storage config (in this case, I am trying to install the storage node) or a joint master/storage node config (here, I am trying to install a sensor node) the install fails with the following error below (full sosetup.log attached).

This is an on-prem install with Internet access, and I installed the OS from the ISO (Centos).  There are no networking issues; the sensor or storage is on the same VLAN as the master.

Any idea what I can do to fix this?

Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/salt/utils/templates.py", line 261, in render_tmpl
    output = render_str(tmplstr, context, tmplpath)
  File "/usr/lib/python3.6/site-packages/salt/utils/templates.py", line 509, in render_jinja_tmpl
    raise SaltRenderError("Jinja variable {}{}".format(exc, out), buf=tmplstr)
salt.exceptions.SaltRenderError: Jinja variable list object has no element 0
[CRITICAL] Rendering SLS 'base:ssl' failed: Jinja variable list object has no element 0
[INFO    ] Fetching file from saltenv 'base', ** done ** 'sensoroni/init.sls'
[INFO    ] Fetching file from saltenv 'base', ** done ** 'telegraf/init.sls'
[INFO    ] Fetching file from saltenv 'base', ** done ** 'healthcheck/init.sls'
[INFO    ] Fetching file from saltenv 'base', ** done ** 'wazuh/init.sls'
[INFO    ] Fetching file from saltenv 'base', ** done ** 'strelka/init.sls'
[INFO    ] Fetching file from saltenv 'base', ** done ** 'strelka/defaults.yaml'
[INFO    ] Fetching file from saltenv 'base', ** done ** 'filebeat/init.sls'
[INFO    ] Fetching file from saltenv 'base', ** done ** 'filebeat/map.jinja'
[INFO    ] Fetching file from saltenv 'base', ** done ** 'fleet/install_package.sls'
[INFO    ] Fetching file from saltenv 'base', ** done ** 'schedule.sls'
[INFO    ] Fetching file from saltenv 'base', ** done ** 'docker_clean/init.sls'
local:
    Data failed to compile:
----------
    Rendering SLS 'base:ssl' failed: Jinja variable list object has no element 0
Errors detected during setup; skipping post-setup steps to allow for analysis of failures.

Thanks,
[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/9456156/sosetup.log)

**Comment by imalc3142** [comment]
Update:  I tried the install (Storage Node failed from freshly installed Master) using 2.3.160, and it failed with the same errors.

**Comment by dougburks** [comment]
After installing the manager, are you making any modifications to it before attempting to join additional nodes?

**Comment by imalc3142** [comment]
The manager and additional nodes are all default installs.

I did get the sensor back up by configuring it as a standalone node.  I am leaving on vacation this morning and will not be working on this again for a week or so.  

Any suggestions would be greatly appreciated.

Thanks,
